### DISCUSSION TITLE
space support modify subdomains or deta support golang

### DISCUSSION THREAD

**Discussion by gitcfly** [deta/cloud-docs#628] [Ideas]
1. Because golang is not supported in deta, I switched to space, but found that custom subdomains are not supported in space, and the default address is too difficult to remember. Can this be released?
2. Or can golang runtime be supported in deta?

**Comment by sudo-subg** [comment]
Once Deta Space is ready for release, it will be merged with Deta Cloud, which will have all the features of Deta Cloud and many more features that Space has to offer. So you will have custom subdomains support on Deta Space soon as the team is currently working on it.
